src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Field.js

```diff
@@ -3,15 +3,18 @@ import React from 'react';
 import {computed} from 'mobx';
 import {observer} from 'mobx-react';
 import log from 'loglevel';
+import jexl from 'jexl';
 import Router from '../../services/Router';
 import {translate} from '../../utils';
-import FieldComponent from '../../components/Form/Field';
+import Form from '../../components/Form';
+import conditionDataProviderRegistry from './registries/conditionDataProviderRegistry';
 import fieldRegistry from './registries/fieldRegistry';
 import fieldStyles from './field.scss';
 import FormInspector from './FormInspector';
 import type {Error, ErrorCollection, SchemaEntry} from './types';
 
 type Props = {|
+    data: Object,
```

Comment:
Here the `Field` component gets the entire data from the form, which allows us to move the evaluation of the conditions into this component, which works quite well. However, there is still the issue with the `Renderer` component being reused in e.g. blocks. These `Renderer` instances get [only the data from the block](https://github.com/sulu/sulu/blob/dc4b04b77af71286cae13aad44d704a83e1755ee/src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js#L264), not from the entire form. So the `data` prop I have introduce here for all the data already exists in the `Renderer`, but it only gets a part of the data, and I would like to have stuff consistent here. Basically two ways are possible for handling this:

1. The `data` prop in this line can be renamed to something like `allData` or `rootData`, then the same name could be used in the `Renderer` component.
2. We change the meaning of the `data` prop of the `Renderer` in order to match it with the one from the `Field`, and rename the current `data` attribute from the `Renderer` to `value`, then it would match the `value` prop of the `Field`.

Reply:
I think i would prefer option 2. Option 1 would mean that the `Renderer` has a `rootData`, `dataPath` and a `data` prop right?

Reply:
Yeah, option 1 would have that consequence.